website/blog/2020-04-22-deployments-made-better.md

```diff
@@ -0,0 +1,68 @@
+---
+title: "Deployment Experience made better!"
+author: Subodh Kumar
+---
+
+Deployment is a process which enables the developers to serve the apps to users. 
```

Comment:
WaveMaker's 1-click deploy feature allows developers deploy their application to AWS, Azure, Google Cloud or Kubernetes cluster

Reply:
Added